packages/circuit-ui/components/List/List.docs.mdx

```diff
@@ -4,42 +4,180 @@ import { Status, Props, Story } from '../../../../.storybook/components';
+
+```html
+<dl>
+  <!-- term and definition -->
+  <dt>Circuit</dt>
+  <dd>A closed, usually circular line that goes around an object or area.</dd>
+  <!-- metadata label and value -->
+  <dt>Account number</dt>
+  <dd>12345</dd>
+  <!-- question and answer -->
+  <dt>How do I find my account number?</dt>
+  <dd>You can find your account number under "Profile details".</dd>
+</dl>
+```
+
+##### Avoid using description lists
+
+Description lists are intentionally not supported by the List component. They are not consistently announced by screen readers, and can confuse users.
+
+For example, consider this simple description list, containing 2 metadata label-value pairs:
+
+<details>
+  <summary>Show the rendered description list</summary>
+  <dl>
+    <dt>Account number</dt>
+    <dd>12345</dd>
+    <dt>Sign-up date</dt>
+    <dd>21.12.2021</dd>
+  </dl>
+</details>
+
+```html
+<dl>
+  <dt>Account number</dt>
+  <dd>12345</dd>
+  <dt>Sign-up date</dt>
+  <dd>21.12.2021</dd>
+</dl>
+```
+
+Here's the screen reader output on three major [screen reader / browser combinations](https://webaim.org/projects/screenreadersurvey9/#browsercombos), when arrowing through the list:
+
+| OS         | Browser | Screen reader | Output                                                                                                     |
+| ---------- | ------- | ------------- | ---------------------------------------------------------------------------------------------------------- |
+| Windows 10 | Chrome  | JAWS          | Definition list of 2 items. Account number. 12345. Sign-up date. 21.12.2021.                               |
+| Windows 10 | Chrome  | NVDA          | List with 4 items. Account number. 12345. Sign-up date. 21.12.2021.                                        |
+| macOS      | Safari  | VoiceOver     | Description list 2 items. Account number, 1 of 4. 12345, 2 of 4. Sign-up date, 3 of 4. 21.12.2021, 4 of 4. |
+
+As you can see, there are inconsistencies between outputs:
+
+- The description list has 2 name-value pairs, but NVDA on Chrome announces it as a "list with 4 items"
+- VoiceOver on macOS Safari correctly announces "2 items", but goes on to announce each item's position on a total of 4 items ("Account number, 1 of 4")
+- JAWS on Chrome announces a "definition list" (the initial name of the `dl` element), although this description list doesn't contain definitions
+- Other screen reader / browser combinations (such as NVDA on Firefox or VoiceOver on Chrome) have yet other behaviors
+
+This can be very confusing to users, so we recommend against using description lists at this time. Instead, use an unordered list or a table.
+
+#### Using lists for other UI elements
+
+In some cases, a list shouldn't only have text as list items, but another UI element such as a card (e.g. for a list of articles on a news website).
+
+The List componnt doesn't support this use case. Instead, use an HTML unordered list element:
```

Comment:
just a small typo, 'component' :)

Reply:
Thanks, fixed in d92018f

Reply:
Had to explicitly enable [cSpell](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for mdx files, this might be useful for you as well 🙂